src/internal/nozzle/gauge_test.go

```diff
@@ -149,6 +151,124 @@ var _ = Describe("Nozzle", func() {
 				},
 			}))
 		})
+
+		It("When Enabled envelopeSelector, matched tags envelope converts the envelope to a Point(s)", func() {
+			tlsServerConfig, tlsClientConfig := buildTLSConfigs()
+
+			streamConnector := newSpyStreamConnector()
```

Comment:
Please review all test cases, if possible put common parts into BeforeEach. and AfterEach, it will keep tests compact.